src/NuGet.Core/NuGet.DependencyResolver.Core/ResolverUtility.cs

```diff
@@ -47,7 +47,11 @@ public static class ResolverUtility
             var currentCacheContext = context.CacheContext;
 
             IList<IRemoteDependencyProvider> remoteDependencyProviders = context.FilterDependencyProvidersForLibrary(libraryRange);
-            LogIfPackageSourceMappingIsEnabled(libraryRange.Name, context, remoteDependencyProviders);
+
+            if (libraryRange.TypeConstraintAllows(LibraryDependencyTarget.Package))
+            {
+                LogIfPackageSourceMappingIsEnabled(libraryRange.Name, context, remoteDependencyProviders);
```

Comment:
There's [another ref](https://github.com/NuGet/NuGet.Client/blob/9187a50daec29061772a2057490697c6c0e4c354/src/NuGet.Core/NuGet.DependencyResolver.Core/ResolverUtility.cs#L240) of this method. Did you check it?

Reply:
Thanks! I missed this

Reply:
Does this issue repro on `packages.config` projects too? If yes, then I can point to where to fix them too.

Reply:
Not sure, can you show me? I can take a look and debug it to see if it is needed

Reply:
Just create `packages.config` projects and use same nuget.config file.